config.gradle

```
@@ -0,0 +1,49 @@
+// Add WPI extensions.
+apply plugin: 'edu.wpi.first.NativeUtils'
+
+nativeUtils.addWpiNativeUtils()
+
+nativeUtils.wpi.addVendor()
+nativeUtils.wpi.vendor.addVendorReposToMaven(true)
+
+nativeUtils {
+    wpi {
+        configureDependencies {
+            wpiVersion = "2023.0.0-alpha-1"
+            niLibVersion = "2022.4.0"
+            opencvVersion = "4.5.5-3"
+            googleTestVersion = "1.11.0-2"
+            imguiVersion = "1.87-2"
+            wpimathVersion = "2023.0.0-alpha-1"
+        }
+    }
+}
+
+nativeUtils.wpi.addWarnings()
+
+if (project.name != 'wpilibcExamples') {
+    nativeUtils.wpi.addReleaseSymbolGeneration()
+}
+
+nativeUtils.setSinglePrintPerPlatform()
+nativeUtils.enableSourceLink()
+
+nativeUtils.platformConfigs.each {
+    if (it.name.contains('windows')) {
+        return
+    }
+    it.linker.args << '-Wl,-rpath,\'$ORIGIN\''
+    if (it.name == 'osxx86-64' || it.name == 'osxarm64') {
+        it.linker.args << "-headerpad_max_install_names"
+    }
```

This makes it match allwpilib's shared/config.gradle.
```suggestion
    if (it.name.contains('osx')) {
        it.linker.args << '-Wl,-rpath,\'@loader_path\''
        it.linker.args << "-headerpad_max_install_names"
    } else {
        it.linker.args << '-Wl,-rpath,\'$ORIGIN\''
    }
```